### PR TITLE
Upgrade AWS

### PR DESCRIPTION
**Pull request by jonyardley**
Looser deps to `aws-sdk` so we can sign AWS ES requests

### CODE REVIEW

**Comment by paulroper** [comment]
Looks like they got rid of the AWS dependency in the Spree core. Getting rid of it here and adding it to our Gemfile might be enough, dunno if we need to update Paperclip or anything too though: https://github.com/spree/spree/commit/af4f3282f0fda56927b069ab30533d8865d19c97

**Comment by paulroper** [comment]
We're running a newer version of Paperclip so I think we could get rid of the dependency entirely here and regression test :tada:

**Comment by jonyardley** [comment]
I'll give it a go 🌮

**Comment by jonyardley** [comment]
Need to do this back in the office. Interwebz is too slow @Fortnum's

**Comment by jonyardley** [comment]
Works for me!

File: core/spree_core.gemspec [hunk 1]
```diff
@@ -20,7 +20,6 @@ Gem::Specification.new do |s|
   s.add_dependency 'activemerchant', '~> 1.44.1'
   s.add_dependency 'acts_as_list', '= 0.3.0'
   s.add_dependency 'awesome_nested_set', '~> 3.0.0.rc.3'
-  s.add_dependency 'aws-sdk', '1.27.0'
```
Comment (jonyardley) [inline core/spree_core.gemspec:23]: 👋

Comment (paulroper) [reply core/spree_core.gemspec:23]: This moves us one line closer to Spree 2.4.0 😄

**Pull request merged**